authority-portal-frontend/src/app/pages/dashboard-page/dashboard-connector-card/dashboard-connector-card.component.ts

```diff
@@ -43,6 +43,7 @@ export class DashboardConnectorCardComponent {
   @HostBinding('class.rounded-xl')
   @HostBinding('class.p-6')
   @HostBinding('class.overflow-hidden')
+  @HostBinding('class.bg-white')
```

Comment:
comment; with all these "bg-white" things im wondering if the bg will ever be part of the theme

Reply:
It would be ugly as hell so I hope not